build.assets/windows/build.ps1

```diff
@@ -89,6 +89,7 @@ function Install-Go {
         [string] $GoVersion
     )
     begin {
+        New-Item -Path "$ToolchainDir" -ItemType Directory -Force | Out-Null
```

Comment:
`Install-Go` and `Install-Node` are called from `dronegen/windows.go`. Perhaps it'd make sense to add another step which would guarantee that `$ToolchainDir` exists rather than doing this from within these functions?

The existence of `$ToolchainDir` seems like a guarantee that should be given to all build steps.

Reply:
I kind of feel like another step for just this one thing would be more cruft than benefit, and it does not really fit in an existing step such as "check out code" either. 🤔 The only steps that use `$ToolchainDir` are `Install-{Go,Node}` and `Enable-{Go,Node}`, others just assume that the toolchains are in PATH (which is updated by `Enable-*`).